labels.md

```diff
@@ -0,0 +1,220 @@
+```
+
+## TiKV
+
+```
+C: Build                        #d1fad7     component/build                     #d1fad7
+C: Build-Time                   #d1fad7     component/build time                #d1fad7
+C: Copr                         #d1fad7     component/coprocessor               #d1fad7
+C: Doc                          #d1fad7     component/docs                      #d1fad7
+C: PD-Client                    #d1fad7     component/PD client                 #d1fad7
+C: Perf                         #d1fad7     component/perf                      #d1fad7
+C: Raft                         #d1fad7     component/raft                      #d1fad7
+C: RocksDB                      #d1fad7     component/RocksDB                   #d1fad7
+C: Server                       #d1fad7     component/server                    #d1fad7
+C: Storage                      #d1fad7     component/storage                   #d1fad7
+C: Test/Bench                   #d1fad7     component/test                      #d1fad7
+                                            component/bench                     #d1fad7
+C: TiKV-Client                  #d1fad7     component/TiKV client               #d1fad7
+C: TiKV-Ctl                     #d1fad7     component/TiKV ctl                  #d1fad7
+C: Titan                        #d1fad7     component/Titan                     #d1fad7
+C: Txn                          #d1fad7     component/transactions              #d1fad7
+C: Util                         #d1fad7     component/util                      #d1fad7
+C: gRPC                         #d1fad7     component/gRPC                      #d1fad7
+D: Easy                         #0e8a16     difficulty/easy                     #0e8a16
+D: Medium                       #f4b169     difficulty/medium                   #0e8a16
+D: Mentor                       #31c639     difficulty/mentor                   #0e8a16
+P: Critical                     #ed0000     priority/high                       #eb6420
+P: High                         #ed8888     priority/medium                     #eb6420
+P: Low                          #eeee00     priority/low                        #eb6420
+P: Release-blocker              #f25c8e     priority/blocker                    #eb6420
+S: BotClose                     #c6054c     status/bot close                    #e6e6e6
+S: CanMerge                     #4be524     status/bot merge                    #e6e6e6
+S: DNM                          #DDDDDD     status/DNM                          #e6e6e6
+S: Discussion                   #fbca04     type/discussion                     #1d76db
+S: Duplicate                    #dddddd     status/closed dup                   #e6e6e6
+S: HelpWanted                   #fbca04     status/help wanted                  #e6e6e6
+S: Invalid                      #dddddd     status/closed invalid               #e6e6e6
+S: LGT1                         #66d7ee     status/waiting on review            #e6e6e6
```

Comment:
Looks like normal PRs should mostly be in a "waiting on review" status and then this label does not have much meaning?